packs/src/tables/wand-of-wonder.json

```diff
@@ -0,0 +1,350 @@
+      ],
+      "drawn": false,
+      "documentCollection": "",
+      "_id": "4XvYA042W3ty8L2C",
+      "text": "An object of the GM's choice disappears into the Ethereal Plane. The object must be neither worn nor carried, within 120 feet of the target, and no larger than 10 feet in any dimension.",
+      "img": "icons/svg/d20-black.svg",
+      "documentId": null,
+      "flags": {}
+    },
+    {
+      "type": 0,
+      "weight": 1,
+      "range": [
+        66,
+        69
+      ],
+      "drawn": false,
+      "documentCollection": "",
+      "_id": "tkTADPmN4rhgB7pH",
+      "text": "You shrink yourself as if you had cast enlarge/reduce on yourself.",
+      "img": "icons/svg/d20-black.svg",
+      "documentId": null,
+      "flags": {}
+    },
+    {
+      "type": 0,
+      "weight": 1,
+      "range": [
+        70,
+        79
+      ],
+      "drawn": false,
+      "documentCollection": "",
+      "_id": "YiBndxEOSit5OROQ",
+      "text": "You cast fireball.",
+      "img": "icons/svg/d20-black.svg",
+      "documentId": null,
+      "flags": {}
+    },
+    {
+      "type": 0,
+      "weight": 1,
+      "range": [
+        80,
+        84
+      ],
+      "drawn": false,
+      "documentCollection": "",
+      "_id": "dmhB1orGtjw0wmac",
+      "text": "You cast invisibility on yourself.",
+      "img": "icons/svg/d20-black.svg",
+      "documentId": null,
+      "flags": {}
+    },
+    {
+      "type": 0,
+      "weight": 1,
+      "range": [
+        85,
+        87
+      ],
+      "drawn": false,
+      "documentCollection": "",
+      "_id": "aAhn7711Uu2EodVB",
+      "text": "Leaves grow from the target. If you chose a point in space as the target, leaves sprout from the creature nearest to that point. Unless they are picked off, the leaves turn brown and fall off after 24 hours.",
+      "img": "icons/svg/d20-black.svg",
+      "documentId": null,
+      "flags": {}
+    },
+    {
+      "type": 0,
+      "weight": 1,
+      "range": [
+        88,
+        90
+      ],
+      "drawn": false,
+      "documentCollection": "",
+      "_id": "94OaXgLFHowfdHFo",
+      "text": "A stream of [[1d4]] × 10 gems, each worth 1 gp, shoots from the wand's tip in a line 30 feet long and 5 feet wide. Each gem deals 1 bludgeoning damage, and the total damage of the gems is divided equally among all creatures in the line.",
+      "img": "icons/svg/d20-black.svg",
+      "documentId": null,
+      "flags": {}
+    },
+    {
+      "type": 0,
+      "weight": 1,
+      "range": [
+        91,
+        95
+      ],
+      "drawn": false,
+      "documentCollection": "",
+      "_id": "BrH4mXjhQZtLJ9qV",
+      "text": "A burst of colorful shimmering light extends from you in a 30‐foot radius. You and each creature in the area that can see must succeed on a DC 15 Constitution saving throw or become blinded for 1 minute. A creature can repeat the saving throw at the end of each of its turns, ending the effect on itself on a success.",
```

Comment:
I think this is probably less important than the spells but we can also link to these conditions like `blinded` (and `petrified` for the 98-100 roll) from the Rules compendium.